control/control.MicroOS.xml

```diff
@@ -263,6 +263,7 @@ textdomain="control"
        </globals>
 	      <software>	
             <default_patterns>microos_base microos_base_packagekit microos_defaults microos_hardware microos_gnome_desktop container_runtime bootloader</default_patterns>
+            <install_recommended config:type="boolean">true</install_recommended>
```

Comment:
well, this element is not known in software. @lslezak what is correct key here?

Reply:
Um, that should be correct: https://github.com/yast/yast-autoinstallation/blob/master/src/autoyast-rnc/software.rnc#L21

Reply:
Ooops, wrong file, it is: https://github.com/yast/yast-installation-control/blob/master/control/control.rnc#L422-L452

Reply:
This is not supported, I think the correct solution is to disable the minimal libzypp config, see https://github.com/yast/skelcd-control-MicroOS/blob/948103a55cad761315ebadba3bc027f6f5a048b8/control/control.MicroOS.xml#L118

Reply:
also would have to undo the `minimalistic_libzypp_config` hack